view/perspective/app.js

```diff
@@ -96,7 +99,7 @@ function handleError(err) {
   }
 
   ERROR_INFO_DIV.innerHTML = msg;
-  u.removeSpinner(SPINNER_ID);
+  removeSpinner();
```

Comment:
reverting? let's make sure we keep any recent changes made since you forked the page

Reply:
oops...
fixed.